src/Components/ResultYes.jsx

```diff
@@ -1,25 +1,36 @@
 import React from "react";
+import Button from "./Button";
+import { useHistory } from "react-router-dom";
 
 const ResultYes = () => {
+  const history = useHistory();
+  const onClick = () => history.push("/");
+
   return (
-    <div>
-      <h2>
+    <>
+      <p class="calc-col title">
         From your answers, it seems that you may be eligible to vacate your
         marijuana misdemeanor conviction!
-      </h2>
-      <p>
-        Your next step is to fill out a Motion and Declaration for Order
-        Vacating Marijuana Conviction.
+      </p>
+      <div className="calc-col results">
+        <p>
+          Your next step is to fill out a Motion and Declaration for Order
+          Vacating Marijuana Conviction.
+        </p>
+
         <a href="https://www.courts.wa.gov/forms/?fa=forms.contribute&formID=38">
```

Comment:
can we hyperlink this? this is not showing up as a hyperlink on the site

Reply:
I'm not sure what you mean by hyperlink? I left it as it's url text because that's how it is on Figma: 
![Screen Shot 2021-06-09 at 10 11 15 AM](https://user-images.githubusercontent.com/58503695/121399338-2d7e3080-c90b-11eb-922e-ea64cb0af775.png)